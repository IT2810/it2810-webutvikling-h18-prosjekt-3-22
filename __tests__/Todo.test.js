import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer'
import Todo from '../src/components/todo.js';

//creates snapshots from todo-component
test('Take snapshot of ToDo', () => {
  const snap = renderer.create( <
    Todo / >
  ).toJSON();
  expect(snap).toMatchSnapshot();
});