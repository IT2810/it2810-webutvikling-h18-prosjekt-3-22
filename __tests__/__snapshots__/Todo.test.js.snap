// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToDo-page test Take snapshot of todo-component 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "paddingBottom": undefined,
      },
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#4d79ff",
        "borderBottomColor": "#ddd",
        "borderBottomWidth": 10,
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontSize": 18,
          "padding": 26,
        }
      }
    >
       Todo 
    </Text>
  </View>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    onSubmitEditing={[Function]}
    placeholder="Add Tasks"
    returnKeyLabel="done"
    returnKeyType="done"
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#252525",
        "borderTopColor": "#ededed",
        "borderTopWidth": 2,
        "color": "#fff",
        "padding": 20,
      }
    }
    testID="addTaskInput"
    value=""
  />
</View>
`;
